packages/core/upload/server/services/metrics.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const { scheduleJob } = require('node-schedule');
+const { FOLDER_MODEL_UID, FILE_MODEL_UID } = require('../constants');
+
+const rand = max => Math.floor(Math.random() * max);
+const getCronRandomHour = () => `${rand(60)} ${rand(60)} ${rand(24)} ${rand(7)} * *`;
```

Comment:
Out of curiosity: what is the benefit of randomizing this? To prevent many requests at the same time from different instances?

Reply:
Yes exactly :)

Reply:
You thought of everything, nice!